package.json

```diff
@@ -30,7 +30,7 @@
     "regen-samples": "pnpm -r run regen-samples",
     "test": "vitest run",
     "test:ci": "vitest run  --coverage --reporter=junit --reporter=default",
-    "test:e2e": "pnpm -r run test:e2e",
+    "test:e2e": "pnpm -r run test:e2e  && pnpm generate-scenarios-summary",
```

Comment:
sorry to go back and forth about this, but now I see maybe why its confusing, this should be specific to the http-specs package no in the whole workspace. This also means removing line 34 in this file. And I do think making it part of the `regen-docs` is actually more accurate as this is what the summary is a doc of the specs

Reply:
@timotheeguerin Sure. I can remove the line 34 `"generate-scenarios-summary": "pnpm -r run generate-scenarios-summary",` from this file. 

and change the script "test:e2e" to `"test:e2e": "pnpm -r run test:e2e  && pnpm regen-docs",` in this file. 

and rename ` generate-scenarios-summary` in the `http-specs` to `regen-docs`. Is that the expectation?

Reply:
Done